tests/rptest/tests/random_node_operations_test.py

```diff
@@ -104,10 +99,10 @@ def _setup_test_scale(self, num_to_upgrade):
 
             self.max_partitions = 32
             self.producer_throughput = 1000 if self.debug_mode else 10000
-            self.node_operations = 10
-            self.msg_size = 128
+            self.node_operations = 3
```

Comment:
subjectively 3 is a bit low... is the test failing often in docker with a higher operation count?

Reply:
i missed it, i changed it to test locally, fixing